### ISSUE TITLE
root package confused with npm package

### ISSUE THREAD

**Issue by mckaymatt** [mbalabash/sdc-check#2]
I installed and ran the tool just like you suggested and it fails every time because it cannot find the version of my app which is the root project. I tried this on both the v1 and v2 versions of my `package-lock.json`.
I think the problem is that `myapp` (i changed the name) is a private package that is not on NPM, but there is a package called `myapp` on NPM. sdc-check doesn't realize that the root package should not be looked up on NPM.
This is true even if i add my app to the `.sdccheckignore`.

```
npm run sdc-check 

> myapp@0.0.7 sdc-check
> sdc-check -d .

⠸ Running sdc-checkERROR: Could not gather metrics
myapp: No matching version found for myapp@0.0.7.
    at module.exports (/tmp/myapp/node_modules/npm-pick-manifest/lib/index.js:209:23)
    at /tmp/myapp/node_modules/pacote/lib/registry.js:126:26
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async getMetrics (file:///tmp/myapp/node_modules/sdc-check/src/metrics.js:28:37)
    at async check (file:///tmp/myapp/node_modules/sdc-check/index.js:41:17)
    at async file:///tmp/myapp/node_modules/sdc-check/src/cli.js:46:14 {
  code: 'ETARGET',
  type: 'version',
  wanted: '0.0.7',
  versions: [
    '1.1.6',  '1.1.7',  '1.1.8',  '1.1.9',  '1.1.10',
    '1.1.11', '1.1.12', '1.1.13', '1.1.14', '1.1.15',
    '1.3.4',  '1.3.5',  '1.3.7',  '1.3.8',  '1.3.9',
    '1.3.10'
  ],
  distTags: { latest: '1.3.10' },
  defaultTag: 'latest'
}
ERROR: Could not perform sdc-check audit
Error: sdc-check internal error
    at file:///tmp/myapp/node_modules/sdc-check/src/cli.js:50:13
    at processTicksAndRejections (node:internal/process/task_queues:96:5)

🚫 sdc-check exited with error
```

**Comment by mbalabash** [comment]
Hey!
Thanks for reporting this issue. 
Yes, you’re right. [This condition](https://github.com/mbalabash/sdc-check/blob/main/src/metrics.js#L18-L23) should be changed to correctly handle this case.
Would you like to create a PR with patch for it?

**Comment by mbalabash** [comment]
Fix released in `sdc-check@1.0.2`. 
Just make sure you have `"private": "true",` in your `package.json` and after upgrade it should work well.
